### PR TITLE
use public builder

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: api/src/ContinuousPipe/Builder/GoogleContainerBuilder/HttpBuildCreator.php [hunk 1]
```diff
@@ -56,7 +56,7 @@ public function startBuild(Artifact $sourceArtifact, string $gcbBuilderVersion):
                     ],
                     'steps' => [
                         [
-                            'name' => 'gcr.io/continuous-pipe-1042/cloud-builder:' . $gcbBuilderVersion,
+                            'name' => 'docker.io/continuouspipe/gcb-builder:' . $gcbBuilderVersion,
```
Comment (sroze) [inline api/src/ContinuousPipe/Builder/GoogleContainerBuilder/HttpBuildCreator.php]: Do we have the correct tags pushed for this?

Comment (OskarStark) [reply api/src/ContinuousPipe/Builder/GoogleContainerBuilder/HttpBuildCreator.php]: yes, `v8` as its defined in the Makefile

Comment (OskarStark) [reply api/src/ContinuousPipe/Builder/GoogleContainerBuilder/HttpBuildCreator.php]: https://hub.docker.com/r/continuouspipe/gcb-builder/tags/

**Comment by OskarStark** [comment]
I prefer #64 instead of this one

**Comment by sroze** [comment]
Agree. Let's close it in favor of #64.

**Pull request closed without merging**